### PR TITLE
feat: adds support to multi-level submodule

### PR DESCRIPTION
**Pull request by amandakarina**
Adds support to sub-modules in structure:
terraform-google-modules/example/step/module/bar-module

### CODE REVIEW

**Comment by apeabody** [comment]
@bharathkkb FYI: This is for compatibility with repo like https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application

File: infra/module-swapper/cmd/testdata/example-with-steps-submodules/examples/example-one/main.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module "test-module" {
+  source  = "terraform-google-modules/example-with-steps-submodules/google"
+  version = "~> 3.2.0"
+
+  project_id   = var.project_id # Replace this with your project ID in quotes
```
Comment (renato-rudnicki) [inline infra/module-swapper/cmd/testdata/example-with-steps-submodules/examples/example-one/main.tf:5]: Based on the comment `# Replace this with your project ID in quotes` I'm wondering if the best approach would be to hardcode the _project_id_ on the main.tf or keep the it as it is (variables) and add the value into a `terraform.tfvars` file.

**Review by renato-rudnicki — COMMENTED** [review]
Only a question, but the rest looks good.

File: infra/module-swapper/cmd/testdata/example-with-steps-submodules/examples/example-one/main.tf (hunk 1 above)
Comment (apeabody) [reply infra/module-swapper/cmd/testdata/example-with-steps-submodules/examples/example-one/main.tf:5]: Thanks @renato-rudnicki - This is testdata, and this particular field isn't actually critical to the test.

**Comment by apeabody** [comment]
@amandakarina - I believe this can be closed?

**Pull request closed without merging**